traffic_portal/app/src/common/modules/form/server/form.server.tpl.html

```diff
@@ -184,8 +187,11 @@
                 </div>
             </div>
             <div class="form-group" ng-class="{'has-error': hasError(serverForm.ip6Address) && hasError(serverForm.ipAddress), 'has-feedback': hasError(serverForm.ip6Address) && hasError(serverForm.ipAddress)}">
-                <label class="control-label col-md-2 col-sm-2 col-xs-12">IPv6 Address</label>
-                <div class="col-md-10 col-sm-10 col-xs-12">
+				<label class="has-tooltip control-label col-md-2 col-sm-2 col-xs-12">IPv6 Address *<div class="helptooltip">
+						<div class="helptext">An IPv6 address and subnet mask of Interface Name.  If the Is Service Address box is checked, then this address will be included in the CrConfig and traffic will be routed to this address. At least one Network IP or IPv6 Address must be provided.</div>
```

Comment:
maybe you should replace Crconfig with 'Traffic Router config' as i think that would make more sense to a normalish user.

Reply:
done